frappe/geo/country_info.json

```diff
@@ -2614,7 +2614,7 @@
   "number_format": "#,###.##",
   "isd": "+963"
  },
- "Taiwan": {
+ "Taiwan, Province of China": {
```

Comment:
Let's not put a country's political position in code 🥴 This can be instead done via India compliance.